cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -100,84 +100,84 @@ This section comprises a glossary of all the keywords -- grouped by category and
-|<<functions-any,any()>>                        | Predicate         | Tests whether a predicate holds for at least one element in the list.
-|<<functions-asin, asin()>>                     | Trigonometric     | Returns the arcsine of the expression.
-|<<functions-atan, atan()>>                     | Trigonometric     | Returns the arctangent of the expression.
-|<<functions-atan2, atan2()>>                   | Trigonometric     | Returns the arctangent2 of a set of coordinates.
-|<<functions-avg,avg()>>                        | Aggregating       | Returns the average of a numeric column.
-|<<functions-ceil, ceil()>>                     | Numeric           | Returns the smallest integer greater than or equal to the argument.
-|<<functions-coalesce,coalesce()>>              | Scalar            | Returns the first non-`null` value in the list of expressions passed to it.
-|<<functions-collect,collect()>>                | Aggregating       | Returns a list containing all collected values.
-|<<functions-cos, cos()>>                       | Trigonometric     | Returns the cosine of the expression.
-|<<functions-cot, cot()>>                       | Trigonometric     | Returns the cotangent of the expression.
-|<<functions-count,count()>>                    | Aggregating       | Returns the number of rows.
+|<<functions-acos, acos()>>                     | Trigonometric     | Returns the arccosine of a number in radians.
+|<<functions-all,all()>>                        | Predicate         | Tests whether a predicate holds for all elements in a list.
+|<<functions-any,any()>>                        | Predicate         | Tests whether a predicate holds for at least one element in a list.
+|<<functions-asin, asin()>>                     | Trigonometric     | Returns the arcsine of a number in radians.
+|<<functions-atan, atan()>>                     | Trigonometric     | Returns the arctangent of a number in radians.
+|<<functions-atan2, atan2()>>                   | Trigonometric     | Returns the arctangent2 of a set of coordinates in radians.
+|<<functions-avg,avg()>>                        | Aggregating       | Returns the average of a set of values.
+|<<functions-ceil, ceil()>>                     | Numeric           | Returns the smallest floating point number greater than or equal to a number.
+|<<functions-coalesce,coalesce()>>              | Scalar            | Returns the first non-`null` value in a list of expressions.
+|<<functions-collect,collect()>>                | Aggregating       | Returns a list containing the values returned by an expression.
+|<<functions-cos, cos()>>                       | Trigonometric     | Returns the cosine of a number.
+|<<functions-cot, cot()>>                       | Trigonometric     | Returns the cotangent of a number.
+|<<functions-count,count()>>                    | Aggregating       | Returns the number of values or rows.
```

Comment:
1) Values or rows: this is because `count` can operate in two variants: a) (matching) rows, and b) a set of values. This is explained in more detail in the description of `count`, but I had to keep it "short and sweet" here in the Glossary.

2) I think I originally mentioned `non-null` (and also in other places), but decided to be much more brief here (where a one-sentence line was necessary). However, in the description, there is a  new "Considerations" table, which talks about all the facets pertaining to `null` and `count`.